### DISCUSSION TITLE
15GB will be used?

### DISCUSSION THREAD

**Discussion by Random72IsTaken** [rustdesk/rustdesk#807] [Q&A] [answered]
While installing on Ubuntu using `sudo apt install ~/Downloads/....deb`, I was shocked by a message saying that 15 point something GB of space will used or something!

Is this just a visual bug, or something on my end for leftover dependencies of sort? -I'm new to Linux! XD

**Comment by rustdesk** [comment, marked as answer]
I was confused also. It is not really 15GB. I do not know why Ubuntu has this false warning.

**Reply by Random72IsTaken** [reply]
I got you then. Thanks for confirming that it's **false**!
🌟 software by the way! 🙂

**Reply by MrConorAE** [reply]
(sorry for necroposting, but I found this too)
I think what Ubuntu is telling you is that to install it, it needs this additional software to *be* installed - i.e. these are the dependencies. Nothing will actually be installed unless you don't have them installed already, which is unlikely because they're pretty common dependencies.

The 15GB stat is how big it *would* be *if you had none of those dependencies installed*.
